### ISSUE TITLE
Sprint 3 – Scrum Meeting (14-12-2016) 

### ISSUE THREAD

**Issue by juzplayplay** [mewind/RSDSupermarketPOS#16]
Hello everyone! Can i know how is the yesterday progress?

**Comment by lkc442012** [comment]
Of course, Raymond. As we discussed yesterday, I have finished the calculation of the subtotal for each product the user selected.

**Comment by juzplayplay** [comment]
Thats good! Do you had any problem in developing this part of module?

**Comment by lkc442012** [comment]
Luckily, i was able to finish all the tasks on time. Thanks to Sin Nee and Huy Lin for their help.

**Comment by huylin25** [comment]
You are welcome!

**Comment by juzplayplay** [comment]
Alright, now lets us discuss about the next step. What do you guys think?

**Comment by mewind** [comment]
Hmm, maybe we should now include the discount part in the payment module?

**Comment by lkc442012** [comment]
I agree with Sin Nee. Actually I have an idea about the discount part. First of all, the system should check whether the user is member, staff or non-member. If the user logs in as member or staff, they can get a discount when they pay.

**Comment by juzplayplay** [comment]
This idea is good. Maybe we can use if-else loop to allow different discount rate for different type of users.

**Comment by huylin25** [comment]
Yes, the database should be able to check the user ID based on their log in. Only non-member will not get the discount.

**Comment by lkc442012** [comment]
Okay, thank you for your suggestion.

**Comment by juzplayplay** [comment]
Anything else? Who have more idea to share with us?

**Comment by mewind** [comment]
In my opinion, we should also include the cancel function in the payment module to allow the user to cancel their payment if they change their mind. This can makes the system more user-friendly.

**Comment by huylin25** [comment]
I agree with Sin Nee. It is important for a POS system to provide the "recover from mistake" for the user.

**Comment by lkc442012** [comment]
Okay. I will try to include all those functions that you guys mentioned. Since this part is not much things to be included, maybe 2 or 3 hours are enough for me to complete it. I will update it immediately after I complete the code.

**Comment by juzplayplay** [comment]
Okay, I hope it will not be much problem in the development. If you have any question, you can discuss with us.

**Comment by lkc442012** [comment]
Thank you guys! I will try my best to finish it.

**Comment by juzplayplay** [comment]
You are welcome. If there is no question, we can dismiss the meeting.

**Comment by mewind** [comment]
Alright.
